hydra-node/hydra-node.cabal

```diff
@@ -113,6 +113,7 @@ library
     , cardano-binary
     , cardano-crypto-class
     , cardano-ledger-alonzo
+    , cardano-ledger-alonzo-test
```

Comment:
Need to re-generate materialization because of this.

Reply:
We should re-generate after moving this stuff where we want the mock server it to reside. Definitely not in `Hydra.Chain.Direct`, but question is in another module in the lib or move it to the `tests` component?